adapters/yieldmo/yieldmo.go

```diff
@@ -16,8 +16,18 @@ type YieldmoAdapter struct {
 	endpoint string
 }
 
+type ExtImpBidderYieldmo struct {
+	adapters.ExtImpBidder
+	Data *ExtData `json:"data,omitempty"`
+}
+
+type ExtData struct {
+	PbAdslot string `json:"pbadslot"`
```

Comment:
You added new bidder extension parameter. Do you want to update static/bidder-params/yieldmo.json file as well?

Reply:
Can you please update documentation PR as well? https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#user-documentation

Reply:
Hi Veronika,

Apologies for the delay in replying to your comment. My understanding is that pbadslot is an additional param coming in from the prebid SDK, and prebid server adapters can be updated to read it: https://docs.prebid.org/features/pbAdSlot.html

Since this is not a bidder specific param, I do not think it is necessary to add anything to static/bidder-params/yieldmo.json, or the user documentation. There are a few other adapters which are ingesting this field (grid, pubmatic), and they have not added it to their bidder.json files.

Please let me know if I am misinterpreting something. Thank you.

Reply:
Hi @ym-winston, I apologize, I didn't know about this. Thank you for pointing me to this!